package.json

```diff
@@ -21,7 +21,7 @@
     "dissecozy:aggregation": "yarn run cozy-konnector-dev -m manifest.webapp build/services/aggregation/dissecozy.js",
     "dissecozy:categorize": "yarn run cozy-konnector-dev -m manifest.webapp build/services/categorize/dissecozy.js",
     "dissecozy:contribution": "yarn run cozy-konnector-dev -m manifest.webapp build/services/contribution/dissecozy.js",
-    "populate": "sh ./scripts/populate.sh 10"
+    "populate": "sh ./scripts/populate.sh 10 2 30"
```

Comment:
Since you already have defaults in `populate.sh`, I would suggest to let the possibility to parametrize this. Typically, remove the `10 2 30` part and call the script with `yarn run populate 20 6 300` (or just `yarn populate` to keep the defaults).
You would also need to remove the `if [ $# != 3 ]` in the populate script and add a default for the instances to create